spec/graphql/mutations/save_repository_mutation_spec.rb

```diff
@@ -0,0 +1,65 @@
+# frozen_string_literal: true
+
+require 'spec_helper'
+
+RSpec.describe 'saveRepository mutation' do
+  let!(:repository) { create :repository_compound }
+  let(:repository_data) do
+    repo = build :repository
+    data = repo.values.slice(:description, :content_type, :public_access).
```

Comment:
Make sure that the generated `content_type` is different from `repository.content_type`.